paper.md

```diff
@@ -0,0 +1,104 @@
+
+For these reasons, HOHQMesh was developed to generate all-quadrilateral and extruded hexahedral meshes suitable for use with spectral element methods. HOHQMesh is a direct quadrilateral mesher, which generates quadrilateral elements by the subdivision method of Schneiders [@schneiders2000algorithms] rather than indirectly from a triangular mesh or by curving a low order mesh. It also sizes and curves the elements based on the length scales in the model, rather than try to modify an existing low order mesh.
+
+# Features
+
+HOHQMesh is designed to require minimal input from the user through the use of a control file. The model defines the geometry in terms of an outer and inner boundary curves.
+
+HOHQMesh features include:
+
+- Unstructured all-quadrilateral or hexahedral meshes
+- Isoparametric polynomial boundary approximations of arbitrary order
+- Automatic geometry-guided refinement
+- Optional user specified local refinement
+- Interior boundaries to separate regions of different properties
+- Symmetric mesh generation
+- Hexahedral meshes from extrusion, rotation, and sweeping of a quadrilateral mesh, with or without scaling
+- Bottom topography variations for extruded hexahedral meshes with automatic resolution of topographic features
+
+# Example
+
+In 1959 the Malpasset dam in France failed and flooded the Reyran river valley down to the Mediterranean sea [@hervouet1999malpasset],[@goutal1999malpasset]. Fig \ref{fig:Total} shows a mesh with 2392 fourth order elements generated by HOHQMesh in 0.44s on an Apple MacBook Pro with a 2.3 GHz Quad-Core Intel
+i7. A zoom of the western portion of the mesh is shown in Fig. \ref{fig:zoom}. The geometry model consists only of an outer boundary, which was specified as a cubic spline, and no inner boundaries. Fig. \ref{fig:WaterHeight} shows a spectral element computation of the water heights using the mesh of Fig. \ref{fig:Total} in the package Trixi.jl [@schlottkelakemper2021purely].
+
+![Spectral element mesh for the Reyran river valley including a portion of the Mediterranean Sea\label{fig:Total}](PaperFigs/malpasset-total-skeleton.png)
+
+![Western portion of the Reyran valley mesh\label{fig:zoom}](PaperFigs/malpasset-total-skeleton-zoom.png)
+
+![Spectral element computation of the water heights at 1985s after the break of the Malpasset dam\label{fig:WaterHeight}](PaperFigs/malpasset_time_1985s.png)
+
+# Related Software
+
+Special purpose quad/hex spectral element grid generators for simple geometries are openly available as part of some spectral element solvers. The preprocessor for FLEXI, HOPR, for instance, will generate cartesian boxes and meshes built from combinations of cartesian boxes, cylinders and spheres.
+
+Spectral element solvers that currently can read meshes generated by HOHQMesh include
+
+- FLUXO [@FLUXO]
+- Trixi.jl [@schlottkelakemper2021purely]
+- HORSES3D [@FERRER2023108700]
+
+The preprocessor HOPR [@HOPR] can also read and modify these meshes.
```

Comment:
Last I saw, HOPR could only read the 2-D ISM files; documentation still refers to HOHQMesh as SpecMesh. Should we say

The High Order Preprocessor (HOPR) [@HOPR] can also read and modify quad meshes generated by HOHQMesh.

Reply:
That is true. I will modify this discussion.